sov-modules/sov-modules-impl/accounts/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+mod call;
+mod genesis;
+mod query;
+#[cfg(test)]
+mod tests;
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use sov_modules_api::{Address, Error};
+use sov_modules_macros::ModuleInfo;
+
+#[derive(BorshDeserialize, BorshSerialize, Debug, PartialEq, Copy, Clone)]
+struct Account {
+    pub addr: Address,
+    pub nonce: u64,
+}
+
+#[derive(ModuleInfo)]
+pub struct Accounts<C: sov_modules_api::Context> {
+    #[state]
+    pub(crate) public_keys: sov_state::StateMap<Address, C::PublicKey, C::Storage>,
+
+    #[state]
+    pub(crate) accounts: sov_state::StateMap<C::PublicKey, Account, C::Storage>,
```

Comment:
Question: Do we need this mapping to be stored in state? Or could we get away with just having `accounts: sov_state::StateMap<C::Address, Account, C::Storage>,` and store pub_key directly in the account? 

If so, I think we could just use `accounts.get(address).unwrap_or_default()` instead of having a separate `create_account` function

Reply:
Discussed offline and we will leave it as it is for now